packages/expo-sqlite/src/SQLiteDatabase.ts

```diff
@@ -23,7 +23,7 @@ export class SQLiteDatabase {
   constructor(
     public readonly databasePath: string,
     public readonly options: SQLiteOpenOptions,
-    private readonly nativeDatabase: NativeDatabase
+    public readonly nativeDatabase: NativeDatabase
```

Comment:
need to get the underlying `nativeDatabase` from the static `backupDatabaseAsync` api. since typescript does not actually have truly `#nativeDatabase` protection. people originally can access to the field already. let's just expose as public.